app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ErrorsController < ApplicationController
```

Comment:
Since these are new files, you'll need to add the following to indicate you're licensing your code under our license and that you want to use frozen string literals
```ruby
# frozen_string_literal: true

# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
```